packages/@adobe/spectrum-css-temp/components/searchwithin/index.css

```diff
@@ -11,49 +11,55 @@ governing permissions and limitations under the License.
 */
 
 @import '../commons/index.css';
-
-:root {
+.spectrum-SearchWithin {
   --spectrum-searchwithin-width: 250px;
 
   /* Force override */
   --spectrum-searchwithin-border-radius: 0;
-  --spectrum-searchwithin-dropdown-min-width: 0;
+  --spectrum-searchwithin-picker-min-width: 0;
 }
 
 .spectrum-SearchWithin {
-  width: var(--spectrum-searchwithin-width);
+  inline-size: var(--spectrum-searchwithin-width);
   display: inline-flex;
   position: relative;
+}
 
-  .spectrum-Dropdown {
-    width: auto;
-    min-width: 0;
-  }
+.spectrum-SearchWithin-picker {
+  inline-size: auto;
+  min-inline-size: 0;
+  flex-shrink: 0;
 
-  .spectrum-Dropdown-trigger {
-    border-top-right-radius: var(--spectrum-searchwithin-border-radius);
-    border-bottom-right-radius: var(--spectrum-searchwithin-border-radius);
+  > button {
+    width: unset;
+    contain: unset;
+    justify-content: unset;
+    border-end-start-radius: var(--spectrum-searchwithin-border-radius);
+    border-start-start-radius: var(--spectrum-searchwithin-border-radius);
   }
+}
+
+.spectrum-SearchWithin-searchfield {
+  flex-grow: 1;
+  order: -1;
+  margin-inline-end: calc(
```

Comment:
Why are we hiding the searchfield border instead of the picker border? Also seems like this should be a single line of code.